ipaserver/install/server/replicainstall.py

```diff
@@ -1321,7 +1315,7 @@ def promote_check(installer):
                 os.environ['KRB5CCNAME'] = ccache
 
     installer._ca_enabled = ca_enabled
-    installer._kra_enabled = kra_enabled
+    installer._kra_enabled = options.setup_kra
```

Comment:
This should be: installer.setup_kra as well and any use of _kra_enabled fixed through the code.
Keep in mind that the variable "installer" here ends up being "options" in the install() function, which is very confusing I know, we should fix that too eventually.